controllers/common.go

```diff
@@ -54,6 +56,17 @@ func newMetadataEnvVar(name, key string) corev1.EnvVar {
 	}
 }
 
+func IsPaused(annotations map[string]string) bool {
+	if annotations == nil {
+		return false
+	}
+	if annotations[AnnotationSearchPause] != "" &&
```

Comment:
Do you need to check for `annotations[AnnotationSearchPause] != nil` ?

Reply:
No , we will get a false returned in that case.